wdl-lint/tests/lints/blank-lines-between-elements/source.errors

```diff
@@ -0,0 +1,217 @@
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:47:11
+   │  
+47 │   task bar {
+   │ ╭──────────^
+48 │ │ 
+49 │ │     meta {
+   │ ╰────^
+   │  
+   = fix: remove the blank line(s)
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:49:11
+   │  
+49 │       meta {
+   │ ╭──────────^
+50 │ │         
+51 │ │         description: "bar"
+   │ ╰────────^
+   │  
+   = fix: remove the blank line(s)
+
+warning[Whitespace]: line contains only whitespace
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:50:1
+   │
+50 │         
+   │ ^^^^^^^^
+   │
+   = fix: remove the whitespace from this line
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:51:27
+   │  
+51 │           description: "bar"
+   │ ╭──────────────────────────^
+52 │ │ 
+53 │ │         outputs: {
+   │ ╰────────^
+   │  
+   = fix: remove the blank line(s)
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:54:19
+   │  
+54 │               u: "u"
+   │ ╭──────────────────^
+55 │ │ 
+56 │ │         }
+   │ ╰────────^
+   │  
+   = fix: remove the blank line(s)
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:60:27
+   │  
+60 │           String s = "hello"
+   │ ╭──────────────────────────^
+61 │ │ 
+62 │ │         String? t
+   │ ╰────────^
+   │  
+   = fix: remove the blank line(s)
+
+warning[Whitespace]: more than one blank line in a row
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:78:1
+   │  
+78 │ ╭ 
+79 │ │     input {}
+   │ ╰────^
+   │  
+   = fix: remove the unnecessary blank lines
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:85:14
+   │  
+85 │       runtime {
+   │ ╭─────────────^
+86 │ │ 
+87 │ │         disks: "50 GB"
+   │ ╰────────^
+   │  
+   = fix: remove the blank line(s)
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:88:23
+   │  
+88 │           memory: "4 GB"
+   │ ╭──────────────────────^
+89 │ │ 
+90 │ │         container: "ubuntu:latest"
+   │ ╰────────^
+   │  
+   = fix: remove the blank line(s)
+
+note[BlankLinesBetweenElements]: extra blank line(s) found
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:90:35
+   │  
+90 │           container: "ubuntu:latest"
+   │ ╭──────────────────────────────────^
+91 │ │         
+92 │ │     }
+   │ ╰────^
+   │  
+   = fix: remove the blank line(s)
+
+warning[Whitespace]: line contains only whitespace
+   ┌─ tests/lints/blank-lines-between-elements/source.wdl:91:1
+   │
+91 │         
+   │ ^^^^^^^^
+   │
+   = fix: remove the whitespace from this line
+
```

Comment:
@a-frantz  - This is another duplication between the rules. I'm not sure why `Whitespace` doesn't flag line 89 if it is flagging this one.

Reply:
The GitHub interface isn't displaying whitespace for me properly. I think there are some lines with wonky whitespace, and the `Whitespace` rule is performing correctly. Can you please try and clean up the `Whitespace` lints that aren't related to the `BlankLinesBetweenElements` rule? As per the other thread, I think long term we want our test cases to be largely compliant with unrelated lints, as opposed to `except`ing them.

Either GitHub is acting up or the `Whitespace` rule is wildly inaccurate. I think it's the former but lmk if not lol.

Reply:
Ah, line 91 has whitespace on it. It isn't an empty line.